### ISSUE TITLE
No preview of some images / url displayed wrong

### ISSUE THREAD

**Issue by meaz** [kriztan/Pix-Art-Messenger#311]
#### General information

* **Version:** 2.1.5
* **Device:** Samsung Galaxy S4 mini
* **Android Version:** LineageOS 15.1

#### Steps to reproduce

Open a muc

#### Expected result

See all the photos sent by participants

#### Actual result

Some photos are not visible: instead, the participants messages contain the url to the photos.
These url are sometime wrong in Pix-Art: for example, `https://myserver/upload/IMG-58473.jpg` becomes  `https://myserver/upload/img-58473.jpg` (IMG -> img). Another example: `https://myserver/upload/uLx3TzMDQb-p810Qtaz9BQ.jpg` became `https://myserver/upload/ulx3tzmdqb-p810qtaz9bq.jpg` in Pix-Art
If I open the same MUC with Conversations, I do see the images, and if I check (still in Conversations) the url of the images that were not visible in Pix-Art, I do get the right url `https://myserver/upload/IMG-58473.jpg`

**Comment by kriztan** [comment]
Please check your version, in 2.1.5 everything was fine, just the 2.2.0 beta from 9th February has faulty links

**Comment by meaz** [comment]
Right, I have the 2.2.0 beta (2019-02-09) version... I just tested the 2.2.0 beta (2019-02-10) version, it seems to work. Sorry for my mistake.

**Comment by kriztan** [comment]
Thanks for your feedback, closing this.

**Comment by ildar** [comment]
Some leftovers (maybe): https://github.com/kriztan/Pix-Art-Messenger/blob/master/src/main/java/eu/siacs/conversations/ui/adapter/MessageAdapter.java#L807

**Comment by ildar** [comment]
I'm sorry, am I getting it right? Previews work only for images sent as
XEP-s. But URL preview only shows page's <TITLE/> ?
Or URLs have some advanced preview also?

**Comment by kriztan** [comment]
You are right @ildar inline images (files) must follow the XEP. The url preview is an other function, this shows a title, description and image if available.

**Comment by ildar** [comment]
Ah! well. I couldn't find any image URL that would be shown. E.g. just sent
this text:
> https://smiles.kolobanga.ru/hd_koloboks/hd_light_skin/kiss3.gif

but it didn't show up as image. Would you please check?
Also, must show image category is Hangouts stickers, e.g.
>
https://lh3.googleusercontent.com/-mzo4vMsngdw/VIDgjqRNQxI/AAAAAAAABfs/krYrvzbmalQ3Umw42HdeUxPgIoH3E2wWgCK8BGAYYCw/s0/FatCat_Impatient.png

**Comment by kriztan** [comment]
It's not intended to display an image URL as an inline-image. Further information about the link preview can be found here: https://ogp.me/
